controllers/configmap_controller.go

```diff
@@ -471,3 +474,18 @@ func (r *ConfigMapReconciler) isKubeAPIServerServingCAConfigMap(obj client.Objec
 	return obj.GetNamespace() == certificates.KubeApiServerOperatorNamespace &&
 		obj.GetName() == certificates.KubeAPIServerServingCAConfigMapName
 }
+
+// generateServiceManifest returns the expected state of the Windows service configmap
+func generateServiceManifest() (*servicescm.Data, error) {
+	services := &[]servicescm.Service{{
+		Name:                         windows.WindowsExporterServiceName,
+		Command:                      windows.WindowsExporterServiceCommand,
+		NodeVariablesInCommand:       nil,
+		PowershellVariablesInCommand: nil,
+		Dependencies:                 nil,
+		Bootstrap:                    false,
+		Priority:                     1,
+	}}
+	files := &[]servicescm.FileInfo{}
```

Comment:
Sorry didn't catch this last time. Shouldn't the Windows exporter binary be added here?

Reply:
All payload files can be added, adding a TODO with a link to the appropriate story.